src/main/java/com/scalar/db/service/AdminModule.java

```diff
@@ -0,0 +1,40 @@
+package com.scalar.db.service;
```

Comment:
I'm wondering if we need another module for Admin that is identical to StorageModule.
Is there any problem updating StorageModule just by adding the line?
```
bind(DistributedStorageAdmin.class).to(config.getAdminClass()).in(Singleton.class);
```

Reply:
Thank you for reviewing this!

> I'm wondering if we need another module for Admin that is identical to StorageModule.

Yes, it's better!  I will fix it in that way. Thanks.